docs/params.ipynb

```diff
@@ -0,0 +1,638 @@
+{
```

Comment:
<p>Just an idea: This felt like it was going in the direction of the Rabi oscillation example mentioned at the beginning. Do you happen to have a super short and simple example that uses expression parameters for a tiny Rabi circuit? If not I can work on adding this later. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I don't have a great example related to the Rabi circuit.  The typical place one sees something like this is in a linear shift of a paramter., maybe that is the one to add?</p>